src/Servers/Kestrel/shared/test/TestCertificates/https-dsa.key

```diff
@@ -0,0 +1,23 @@
+-----BEGIN DSA PRIVATE KEY-----
+Proc-Type: 4,ENCRYPTED
+DEK-Info: DES-EDE3-CBC,25877371F267513A
```

Comment:
We don't support this style of encryption, our PEM support is limited to RFC 7468 PEM, which forbids attributes.

Reply:
What is the user experience when loading an unsupported key?

We want to give a good error message On startup if the key won’t work

Reply:
Thanks, can I just strip the attributes? I generated these with openssl, I'll try to generate this one without attributes

Reply:
> What is the user experience when loading an unsupported key?

We throw, telling you that either we couldn't find the key or that it is not in a valid format. I have a test to cover that.

Reply:
> can I just strip the attributes?

No, there are two forms of making an encrypted private key:

1) Build the plaintext key, encrypt it and stuff it into an encrypted key envelope (then use the PEM header type of `-----BEGIN ENCRYPTED PRIVATE KEY-----`)
2) Build the plaintext key, encrypt it and write down that you encrypted it using attributes (keep the PEM header type of `-----BEGIN PRIVATE KEY-----`).

Theoretically you could stuff the payload here into the PKCS#8 EncryptedPrivateKeyInfo format and change the header, but it's a sort of annoying amount of work.

If the openssl CLI isn't doing a friendly job, you can always use a plaintext key, use dsa.ImportFromPem, then dsa.ExportEncryptedPkcs8PrivateKey and write the output as PEM.  ($"-----BEGIN ENCRYPTED PRIVATE KEY-----{Environment.NewLine}{Convert.ToBase64String(data, somethingAboutNewlines)}{Environment.NewLine}-----END ENCRYPTED PRIVATE KEY-----").

Reply:
Yeah, I figured this morning. I have an update that I'm sending where i've created them in pkcs8 format.

Reply:
I've updated all the tests to use pkcs8 keys.